packages/dynamodb-streams-readable/test/index.js

```diff
@@ -209,30 +203,30 @@ test.serial('reads latest records', async t => {
 
   let count = 0;
   return Promise.all([
-    new Promise((resolve, reject) =>
+    new Promise((resolve, reject) => {
       readable
-        .on('data', function(recordSet) {
+        .on('data', function (recordSet) {
           recordSet.forEach(record => {
             t.deepEqual(record.dynamodb.Keys.Id, subsequentDocuments[count].Item.Id);
             count = count + 1;
           });
           if (count > subsequentDocuments.length) t.fail('should not read extra records');
           if (count === subsequentDocuments.length) readable.close();
         })
-        .on('end', function() {
+        .on('end', function () {
           t.deepEqual(
             count,
             subsequentDocuments.length,
             `read ${subsequentDocuments.length} records`
           );
           resolve();
         })
-        .on('error', function(err) {
+        .on('error', function (err) {
           t.fail('should not error');
           reject(err);
-        })
-    ),
-    wait(100).then(() => batchWriteItem(dynamodb, tableName, subsequentDocuments))
+        });
+    }),
+    delay(100).then(() => batchWriteItem(dynamodb, tableName, subsequentDocuments))
```

Comment:
next change will be `defer` ? 😆 just kidding

Reply:
`deferBecauseFaustPrefersIt` :trollface: